src/mpyl/steps/deploy/bpm/__init__.py

```diff
@@ -0,0 +1,73 @@
+"""Bpm deployment related helper methods"""
+
+from logging import Logger
+from python_on_whales import DockerException
+from ..bpm.camunda_modeler_client import CamundaModelerClient
+from ..bpm.cluster import deploy_diagram_to_cluster, get_docker_container
+from ..bpm.modeler import deploy_diagram_to_modeler
+from ....utilities.bpm import CamundaConfig
+from ....utilities.http_client.exceptions import HTTPRequestError, AuthorizationError
+from ...models import Output
+
+
+def deploy_to_cluster(
+    logger: Logger, project_name: str, config: CamundaConfig
+) -> Output:
+    docker_container = get_docker_container(config)
+    try:
+        deploy_diagram_to_cluster(logger, config, docker_container)
+    except DockerException:
+        return Output(
+            success=False,
+            message=f"Deploy BPMN diagrams to cluster for project {project_name} have one or more failures",
+            produced_artifact=None,
+        )
+    finally:
+        docker_container.stop()
+        docker_container.remove()
+
+    return Output(
+        success=True,
+        message=f"Deployed all diagrams in {project_name} to Camunda cluster",
+        produced_artifact=None,
+    )
+
+
+def deploy_to_modeler(
+    logger: Logger, project_name: str, config: CamundaConfig
+) -> Output:
+    credentials = config.modeler_credentials.__dict__
```

Comment:
@koeves is referring to `__dict__` okay?

Reply:
This `__dict__` thing can print the class structure as a json object. Not sure if this is a good practice

Reply:
Oh, no, then I would use the `json` package:

```python
import json
credentials = json.dumps(config.modeler_credentials)
```

Reply:
sorry I was wrong __dict__ is to convert the class to a dictionary, but I do understand it's not a good practive. I got an error: `Object of type CamundaModelerCredentials is not JSON serializable. ` while using json.dumps. From there I realized that I only need a dictionary not json. So now I added a function in the class to create a dictionary structure